tide_webform.module

```diff
@@ -28,6 +28,94 @@ function tide_webform_form_alter(&$form, FormStateInterface $form_state, $form_i
   if (!empty($form['#webform_id']) && $form['#webform_id'] == 'tide_webform_content_rating') {
     $form['#attached']['library'][] = 'tide_webform/content_rating';
   }
+
+  if ($form_id == 'webform_ui_element_form') {
+    $form['#after_build'][] = 'tide_webform_webform_ui_element_form_after_build';
+  }
+}
+
+/**
+ * After_build callback.
+ *
+ * @param array $form
+ *   The form.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   Form state.
+ *
+ * @return array
+ *   The form.
+ */
+function tide_webform_webform_ui_element_form_after_build(array $form, FormStateInterface $form_state) {
+  $storage = $form_state->getStorage();
+  // Unset the default properties of the Privacy Statement element to force
```

Comment:
Can you explain why this is required @sonnykt? What happens if you don't do this?

Reply:
to force webform module to export those default values to YAML. Webform does not export default values in its source YAML.